.github/workflows/tests.yml

```diff
@@ -159,32 +163,45 @@ jobs:
       with:
         distribution: 'temurin'
         java-version: ${{ matrix.java-version }}
-    - name: Set up Gradle 7.3 # must be done manually due to no gradle build in capa
+    - name: Set up Gradle ${{ matrix.gradle-version }} # must be done manually due to no gradle build in capa
       run: |
         mkdir /opt/gradle
-        wget "https://services.gradle.org/distributions/gradle-7.3-bin.zip" -O /opt/gradle/gradle-7.3.zip
-        unzip /opt/gradle/gradle-7.3.zip -d /opt/gradle
-    - name: Install Ghidra 10.3
+        wget "https://services.gradle.org/distributions/gradle-${{ matrix.gradle-version }}-bin.zip" -O /opt/gradle/gradle-7.3.zip
+        unzip /opt/gradle/gradle-${{ matrix.gradle-version }}.zip -d /opt/gradle
+    - name: Install Ghidra ${{ matrix.ghidra-version }} 
       run: |
         mkdir ./.github/ghidra
-        wget "https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_10.3_build/ghidra_10.3_PUBLIC_20230510.zip" -O ./.github/ghidra/ghidra_10.3_PUBLIC.zip
-        unzip .github/ghidra/ghidra_10.3_PUBLIC.zip -d .github/ghidra/
-    - name: Install Jep 4.1.1
+        mkdir ./.github/ghidra/project
+        wget "https://github.com/NationalSecurityAgency/ghidra/releases/download/Ghidra_${{ matrix.ghidra-version }}_build/ghidra_${{ matrix.ghidra-version }}_PUBLIC_20230510.zip" -O ./.github/ghidra/ghidra_${{ matrix.ghidra-version }}_PUBLIC.zip
```

Comment:
`...PUBLIC_20230510...` is going to change with each release as it's the date when the build was completed (e.g. https://github.com/NationalSecurityAgency/ghidra/releases/tag/Ghidra_10.3.1_build) so we'll need to include this in the hard-coded Ghidra version.

Reply:
I've included a `build-version` variable in the matrix. I also made sure that the url string remains the same across builds. Hopefully we can continue to pull different/multiple versions in this manner. Please let me know if any changes should be made.